playbooks/appsemblerPlaybooks/ficus_enterprise.yml

```diff
@@ -137,3 +137,12 @@
     - role: postgresql_aggregation
       when: COMMON_ENVIRONMENT == "prod"
       tags: ['postgresql_aggregation_role']
+
+- name: Install Appsembler Reporting
+  hosts: "edxapp-server-1"
+  sudo: True
+  gather_facts: True
+  when: 
+    - "EDXAPP_APPSEMBLER_FEATURES is defined and EDXAPP_APPSEMBLER_FEATURES.get('ENABLE_APPSEMBLER_REPORTING', False)"
+    - "COMMON_ENVIRONMENT == 'prod'"
+  tags: ['appsembler_reporting_role']
```

Comment:
This doesn't look right. Please double check, as there's no `roles` nor `tasks`.